packages/checkout-ui-extensions/src/components/Grid/Grid.ts

```diff
@@ -2,25 +2,29 @@ import {createRemoteComponent} from '@remote-ui/core';
 
 import {
   ViewLikeAccessibilityRole,
-  Unit,
   Spacing,
   InlineAlignment,
   BlockAlignment,
 } from '../shared';
 
-export type GridItemSize = 'auto' | 'fill' | number | Unit<'fr' | '%'>;
+/* eslint-disable eslint-comments/no-unlimited-disable */
+// eslint-disable-next-line
+export type GridItemSize = 'auto' | 'fill' | number | `${number}fr` | `${number}%`;
+/* eslint-enable eslint-comments/no-unlimited-disable */
```

Comment:
Why's eslint disabled here?

Reply:
There's a mismatch of version of Prettier between checkout-web and ui-extensions. In ui-extensions, its a version that throw errors for template literals. This is out of scope of this PR and since it affects more than our package, I believe it would be preferable to do this separately.